README.md

```diff
@@ -1 +1,62 @@
+# API Reference 2.0
 
+##Introduction
+The Open API 2.0 is the new version of the publicly available protobuf-based API developed by Spotware. It allows third-party service providers to integrate additional tools and applications for trading and analysis with the data and functionality from Spotware platform by getting all the required cTID data, market data and performing all possible trading operations on behalf of other cTrader users.
+
+##Documentation
+Full documentation can be found on [Spotware Connect website](https://connect.spotware.com/docs/open_api_2)
+
+## How to start
+1. Download this project.
+2. Import the project to the chosen IDE as a Maven project.
+4. Run command `mvn install` to download libraries and compile Protocol Buffer classes.
+0. Register your application as described [here](https://connect.spotware.com/docs/open_api_2/getting_started_v2). Set `CLIENT_PUBLIC_ID` and `CLIENT_SECRET`
+ in [application.properties](./openapi2-example/src/main/resources/application.properties). 
+0. Login to [Demo cTrader](https://ct.spotware.com/) to create an account.
+0. Generate a token as decribed [here](https://connect.spotware.com/docs/open_api_2/getting_started_v2/open_authentication_v2). Set `ACCESS_TOKEN` and 
+`CTID_TRADER_ACCOUNT_ID` in [application.properties](./openapi2-example/src/main/resources/application.properties).
+5. Run examples from `openapi2-example` submodule.
+
+## Suggested Popular Development Platform IDE
+1. [Eclipse](https://www.eclipse.org/downloads/)
+2. [Spring Tool Suite](https://spring.io/tools3/sts/all)
+3. [IntelliJ IDEA](https://www.jetbrains.com/idea/download/)
+
+## Basic used libraries
+1. Google Protocol Buffers (Protobuf). [Documentation reference](https://connect.spotware.com/docs/open_api_2/introduction_to_protocol_buffers_v2). 
+[Official site](https://developers.google.com/protocol-buffers/).
+2. Netty. [Official User Guide](https://netty.io/wiki/user-guide-for-4.x.html). [Official site](https://netty.io/).
+
+## Content of the project
+#### Submodule `openapi2-client`
+This is a main codebase to open a channel to the server and send requests in the Protobuf format.
+Protocol files are located in [protobuf](./openapi2-client/src/main/protobuf) directory. 
+
+_Please note that Protocol Buffer classes (from the package `com.xtrader.protocol.openapi.v2`) are generated by the Maven plugin.
+So in a just downloaded project `java` files in this package are absent._
+
+#### Submodule `openapi2-example`
+Several examples to send often used requests:
+- authorization
+- assets list
+- symbols list
+- find symbol by ID
+- subscribe on price spots
+- process Symbol Changed Event
+- get symbol conversion chain
+
+File [application.properties](./openapi2-example/src/main/resources/application.properties) contains properties to be set:
+- `host` You can found it on [Spotware Proxy Cloud page](https://connect.spotware.com/docs/open_api_2/proxy_cloud_v2).
+Default demo host is `demo.ctraderapi.com`
```

Comment:
Let's put an additional link to our "Spotware Proxy Cloud" page in documentation on this line.